usage/conf/usage.conf

```diff
@@ -0,0 +1,28 @@
+env USER=media-service
+env USER_HOME=/home/media-service
+env JAR=/home/media-service/app.jar
+
+# Workaround for https://github.com/aws/aws-sdk-java/issues/123
+env JAVA_OPTS="-XX:MaxRAMFraction=2 -XX:InitialRAMFraction=2 -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:gc.log"
```

Comment:
Just curious how and why this differs from the other "conf" run scripts? e.g. no `-Dfile.encoding=UTF-8` etc?